linchpin/cli/__init__.py

```diff
@@ -674,7 +678,7 @@ def _build(self, pf, pf_data=None):
 
 
     def _execute(self, provision_data, targets,
-                 action='up', run_id=None, tx_id=None):
+                 action='up', run_id=None, tx_id=None, vault_pass=''):
```

Comment:
As demonstrated, this variable is basically being passed around. The context variable is there for that purpose, use it instead. See the reference above for `do_action`.